rsc/src/main/scala/rsc/scan/Scanner.scala

```diff
@@ -261,8 +261,9 @@ final class Scanner private (
       nextChar()
     }
     val parsee = {
-      if (lexeme.startsWith("-")) "-" + lexeme.substring(3)
-      else lexeme.substring(2)
+      val lexemeVal = lexeme
```

Comment:
How about we say `val lexeme = this.lexeme` here to avoid the need to think about names every time we need to cache lexeme?

Reply:
Thats a better name I agree.